rhizome/host/bin/delete-old-serial-logs

```diff
@@ -0,0 +1,24 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require "fileutils"
+
+directory_path = "/var/log/ubicloud/serials"
+
+# Delete files older than 1 hour
+Dir.glob(File.join(directory_path, "*")).each do |file|
+  next unless File.file?(file) # Skip directories
+
+  file_age_in_hours = (Time.now - File.mtime(file)) / 3600
+  FileUtils.rm(file) if file_age_in_hours > 1
```

Comment:
Customers often delay opening support tickets. Starting to debug an issue within one hour might be too hasty. Given 1GB directory limit, could we possibly extend this limit to 24 hours?